cms/static/coffee/spec/models/textbook_spec.coffee

```diff
@@ -2,9 +2,14 @@ define ["backbone", "js/models/textbook", "js/collections/textbook", "js/models/
 (Backbone, Textbook, TextbookSet, Chapter, ChapterSet, main) ->
 
     beforeEach ->
-        @addMatchers
-            toBeInstanceOf: (expected) ->
-                return @actual instanceof expected
+        jasmine.addMatchers
+            toBeInstanceOf: () ->
+                return {
+                    compare: (actual, expected) ->
+                        return {
+                            pass: actual instanceof expected
+                        }
```

Comment:
Seems a generally useful assert. We should move this to `jasmine-extensions.js`.